pkgs/tools/text/rsbkb/default.nix

```diff
@@ -6,16 +6,19 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "rsbkb";
-  version = "1.4";
+  version = "1.4_823b35";
```

Comment:
The Nixpkgs convention is `1.4-unstable-YYYY-MM-DD` with the commit date. (But if you move to `cargoPatches` that won’t be relevant.)